src/transaction/log_replication_atomic.hpp

```diff
@@ -91,8 +91,11 @@ namespace cublog
 
     private:
       atomic_replication_helper m_atomic_helper;
-      log_lsa m_lowest_unapplied_lsa;
+
       log_lsa m_processed_lsa = NULL_LSA; /* protected by m_redo_lsa_mutex with m_redo_lsa */
```

Comment:
The comment should be revised along. I guess we can just delete it.

Reply:
removed